mulog-core/src/com/brunobonacci/mulog.clj

```diff
@@ -269,14 +270,34 @@ For more information, please visit: https://github.com/BrunoBonacci/mulog
   ```
 
   The local context can be nested and ti will be inherited by
-  all the **μ/log** calls within nested functions as long as they
+  all the ***μ/log*** calls within nested functions as long as they
   are in the same execution thread and which the scope of the block.
   "
   [context & body]
   `(binding [*local-context* (merge *local-context* ~context)]
      ~@body))
 
 
+(defmacro verbose [event-name & pairs]
+  `(with-context {:mulog/level ::lvl/verbose}
+     (log ~event-name ~@pairs)))
+(defmacro debug [event-name & pairs]
+  `(with-context {:mulog/level ::lvl/debug}
+     (log ~event-name ~@pairs)))
+(defmacro info [event-name & pairs]
+  `(with-context {:mulog/level ::lvl/info}
+     (log ~event-name ~@pairs)))
+(defmacro warning [event-name & pairs]
+  `(with-context {:mulog/level ::lvl/warning}
+     (log ~event-name ~@pairs)))
+(defmacro error [event-name & pairs]
+  `(with-context {:mulog/level ::lvl/error}
+     (log ~event-name ~@pairs)))
+(defmacro fatal [event-name & pairs]
+  `(with-context {:mulog/level ::lvl/fatal}
+     (log ~event-name ~@pairs)))
```

Comment:
if the `{:mulog/level ::lvl/fatal}` only applies to that one event then it is better to just add it as a `pairs`

``` clojure
(defmacro fatal [event-name & pairs]
  `(log ~event-name :mulog/level ::lvl/fatal ~@pairs))
```
the  *context* is something you want to apply to all events within the scope

Reply:
True that! Thanks.